header.php

```diff
@@ -24,31 +24,42 @@
 <div id="page" class="site">
 	<a class="skip-link screen-reader-text" href="#content"><?php esc_html_e( 'Skip to content', 'siteorigin-corp' ); ?></a>
 
-	<header id="masthead" class="site-header" role="banner">
+	<?php if ( class_exists( 'Woocommerce' ) && is_store_notice_showing() ) : ?>
+		<div id="topbar">
+			<?php siteorigin_corp_woocommerce_demo_store(); ?>
+		</div><!-- #topbar -->
+	<?php endif; ?>
+	<header id="masthead" class="site-header<?php if ( siteorigin_setting( 'header_sticky' ) ) echo ' sticky'; ?>" <?php if ( siteorigin_setting( 'header_scales' ) ) echo 'data-scale-logo="true"' ?> >
 
 		<div class="corp-container">
 
 			<div class="site-header-inner">
 
 				<div class="site-branding">
 					<?php siteorigin_corp_display_logo(); ?>
-					<?php if ( siteorigin_setting( 'branding_site_description' ) ) : ?>
+					<?php if ( siteorigin_setting( 'header_site_description' ) ) : ?>
 						<p class="site-description"><?php bloginfo( 'description' ); ?></p>
 					<?php endif ?>
 				</div><!-- .site-branding -->
 
-				<nav id="site-navigation" class="main-navigation" role="navigation">
+				<nav id="site-navigation" class="main-navigation">
 
-					<?php wp_nav_menu( array( 'theme_location' => 'menu-1', 'menu_id' => 'primary-menu' ) ); ?>
+					<?php $mega_menu_active = function_exists( 'ubermenu' ) || function_exists( 'max_mega_menu_is_enabled' ) && max_mega_menu_is_enabled( 'menu-1' ); ?>
 
-	                <button id="search-button" class="search-toggle">
-	                    <span class="open"><?php siteorigin_corp_display_icon( 'search' ); ?></span>
-                	</button>
+					<?php if ( siteorigin_setting( 'navigation_header_menu' ) ) wp_nav_menu( array( 'theme_location' => 'menu-1', 'menu_id' => 'primary-menu' ) ); ?>
 
-					<a href="#menu" id="mobile-menu-button">
-						<?php siteorigin_corp_display_icon( 'menu' ); ?>
-						<span class="screen-reader-text"><?php esc_html_e( 'Menu', 'siteorigin-corp' ); ?></span>
-					</a>
+					<?php if ( siteorigin_setting( 'navigation_menu_search' ) ) : ?>
```

Comment:
Indenting needs to be cleaned up so opening and closing `if` and `endif` are on the same line.